src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -278,9 +311,13 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
             else {
                 final ParseErrorMessage parseErrorMessage = result.getParseErrorMessage();
-                log(parseErrorMessage.getLineNumber(),
-                        parseErrorMessage.getMessageKey(),
-                        parseErrorMessage.getMessageArguments());
+                final boolean looseHtmlInJavadoc = "javadoc.unclosedHtml"
```

Comment:
I am updating PR with the above changes. We can revert these changes if the approach turns out to be wrong.